api/guides/access_tokens.md

```diff
@@ -1,133 +1,100 @@
-```
-
-These commands will create two files in the current directory _(usually the current user's home directory)_.
+The next screen prompts you to select the product profile(s) to associate with the API integration.
 
 {% alert info, Note %}
-The names of the files may be changed as desired. Keep the private key safe and limit access!
+Product profiles are managed by your organization through the Adobe Admin Console, and contain specific sets of permissions for granular features in Adobe Experience Platform. Product profiles and their permissions can only be managed by users with administrator privileges within your organization. If you are unsure which product profile(s) to select for the API, contact your administrator.
 {% endalert %}
 
-  
-Next, upload the certificate (**crt**) file via the Adobe I/O configuration window by either drag-n-drop or select to upload.
+Select the desired product profile(s) from the list, then select **Save configured API** to complete the API registration.
 
-{% alert info, Note %}
-Multiple certificates may be added.
-{% endalert %}
+![](/images/access-token/select-product-profile.png)
 
-![](/images/access-token/IO-5.png)
+Once the API has been added to the project, the project page reappears on the Experience Platform Launch API page. From here, scroll down to the **Service Account (JWT)** section, which provides the following access credentials that are required in all calls to the Launch API:
 
-Next, we need to assign rights to this Integration.  These integrations share the same permissions that can be assigned to users through the Admin Console.  Any product profiles that exist for Launch will show up in the list at the bottom of the page.  Select the product profile that is appropriate for this integration.
+* **CLIENT ID**: The Client ID is the required `[KEY]` which must be provided in the `x-api-key` header.
+* **ORGANIZATION ID**: The Organization ID is the `[ORG_ID]` value that must be used in the `x-gw-ims-org-id` header.
 
-If you need help creating a new product profile, an Org Admin or a Product Admin for Launch can do this is in the Admin Console where users are managed.  You can read more about this in the user docs [here](https://docs.adobe.com/content/help/en/launch/using/reference/admin/user-permissions.html#n3-create-your-product-profile).
+![](/images/access-token/access-creds.png)
 
-Click **"Create Integration"**.  You should now see the success screen.
+### Authentication for each session
 
-![](/images/access-token/IO-6.png)
+Now that you have your `[KEY]` and `[ORG_ID]` values, the final step is generating a `[TOKEN]` value.
 
 {% alert info, Note %}
-If an error is encountered, it's generally related to certificate generation issues.
+Note that these tokens expire after 24 hours.  If you are using this integration for an application, it is a good idea to obtain your bearer token programmatically from within your application.
```

Comment:
Do we have anything we can link to that would provide guidance or instructions on generating this programmatically within the application?